### PR TITLE
Circle test

### PR DESCRIPTION
**Pull request by effigies**
Building on #85. Need a PR to trigger Circle.

### CODE REVIEW

**Comment by effigies** [comment]
@sappelhoff This is working. Looks like we need to override the base URL config option for Circle. Feel free to cherry-pick or just rename this PR and merge.

**Comment by sappelhoff** [comment]
Thanks for the iterations, but for me there is still something wrong, see the index.html file:

https://15-38337026-gh.circle-artifacts.com/0/html/index.html

the GitHub action is now working, which is nice, but it links to an empty page with "artifact not found".

I find that surprising, because the same setup works on the bids-specification repo.

**Comment by effigies** [comment]
Guess I should have waited for it to render (and also read through the `revert` commit) before commenting.

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+                    bundle
+
+            - run:
+                name: build and output to _site
+                command: |
+                    bundle exec jekyll build --baseurl /0/home/circleci/project/_site
+
+            - store_artifacts:
+                path: ~/project/_site
+                destination: html
```
Comment (sappelhoff) [inline .circleci/config.yml]: mh perhaps this was the culprit, in reverting https://github.com/bids-standard/bids-website/pull/87/commits/499e60f351776f630c606a9fcf88cc6615a01dd2 you (accidentally?) re-introduced the `destination` key ... which overrides the default destination of `circleci/projects`

**Pull request closed without merging**